2025-09-19 10:55:54 [[31merror[39m]: Server error: listen EADDRINUSE: address already in use :::3000
{
  "service": "expense-tracker-api",
  "code": "EADDRINUSE",
  "errno": -48,
  "syscall": "listen",
  "address": "::",
  "port": 3000,
  "stack": "Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1940:16)\n    at listenInCluster (node:net:1997:12)\n    at Server.listen (node:net:2102:7)\n    at Function.listen (/Users/lawrencevaldez/Personal Projects/expensetracker/backend/node_modules/express/lib/application.js:605:24)\n    at startServer (/Users/lawrencevaldez/Personal Projects/expensetracker/backend/dist/server.js:125:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)"
}
2025-09-19 10:56:24 [[31merror[39m]: Server error: listen EADDRINUSE: address already in use :::3000
{
  "service": "expense-tracker-api",
  "code": "EADDRINUSE",
  "errno": -48,
  "syscall": "listen",
  "address": "::",
  "port": 3000,
  "stack": "Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1940:16)\n    at listenInCluster (node:net:1997:12)\n    at Server.listen (node:net:2102:7)\n    at Function.listen (/Users/lawrencevaldez/Personal Projects/expensetracker/backend/node_modules/express/lib/application.js:605:24)\n    at startServer (/Users/lawrencevaldez/Personal Projects/expensetracker/backend/src/server.ts:175:24)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)"
}
2025-09-19 10:56:50 [[31merror[39m]: Server error: listen EADDRINUSE: address already in use :::3000
{
  "service": "expense-tracker-api",
  "code": "EADDRINUSE",
  "errno": -48,
  "syscall": "listen",
  "address": "::",
  "port": 3000,
  "stack": "Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1940:16)\n    at listenInCluster (node:net:1997:12)\n    at Server.listen (node:net:2102:7)\n    at Function.listen (/Users/lawrencevaldez/Personal Projects/expensetracker/backend/node_modules/express/lib/application.js:605:24)\n    at startServer (/Users/lawrencevaldez/Personal Projects/expensetracker/backend/src/server.ts:175:24)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)"
}
2025-09-19 11:17:26 [[31merror[39m]: Authentication failed: Token verification failed
{
  "service": "expense-tracker-api",
  "statusCode": 401,
  "isOperational": true,
  "stack": "Error: Token verification failed\n    at Function.verifyAccessToken (/Users/lawrencevaldez/Personal Projects/expensetracker/backend/src/utils/jwt.ts:63:15)\n    at authenticateToken (/Users/lawrencevaldez/Personal Projects/expensetracker/backend/src/middlewares/auth.ts:40:30)\n    at Layer.handleRequest (/Users/lawrencevaldez/Personal Projects/expensetracker/backend/node_modules/router/lib/layer.js:152:17)\n    at trimPrefix (/Users/lawrencevaldez/Personal Projects/expensetracker/backend/node_modules/router/index.js:342:13)\n    at /Users/lawrencevaldez/Personal Projects/expensetracker/backend/node_modules/router/index.js:297:9\n    at processParams (/Users/lawrencevaldez/Personal Projects/expensetracker/backend/node_modules/router/index.js:582:12)\n    at next (/Users/lawrencevaldez/Personal Projects/expensetracker/backend/node_modules/router/index.js:291:5)\n    at Function.handle (/Users/lawrencevaldez/Personal Projects/expensetracker/backend/node_modules/router/index.js:186:3)\n    at router (/Users/lawrencevaldez/Personal Projects/expensetracker/backend/node_modules/router/index.js:60:12)\n    at Layer.handleRequest (/Users/lawrencevaldez/Personal Projects/expensetracker/backend/node_modules/router/lib/layer.js:152:17)"
}
